### PR TITLE
Add support for iOS full screen web clip, clean up grammar

### PR DESCRIPTION
**Pull request by TheGamer1002**
# Description
This PR adds support for iOS/iPadOS’s [documented Safari features](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html) and cleans up some of the English.
## Type of change

Please delete options that are not relevant.

- [x] Feature addition & English cleanup

# How Has This Been Tested?

__Complete if applicable.__

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [x] N/A yet, will test in a few hours

# Configuration

## Games

- [x] Euro truck simulator 2
- [x] American truck simulator

**Game version:**

- Euro truck simulator 2: [e.g. 1.40.5.13]
- American truck simulator: [e.g. 1.40.5.13]

## Context

__Complete if applicable.__

**Desktop:**

- OS: [e.g. iOS] Windows 11
- Browser [e.g. chrome, safari] N/A
- Version [e.g. 22] N/A

**Smartphone:**

- Device: [e.g. iPhone6] iPad 6 w/ Wifi
- OS: [e.g. iOS8.1] iOS 15
- Browser [e.g. stock browser, safari] Safari
- Version [e.g. 22] Unknown

**Additional information:**

- Skin name: [e.g. JAGFx]
- Dashboard version: [e.g. v1.2.1]
- SCS Telemetry version: [e.g. 1.10.5]
- Node version: [e.g. node14]
Irr.
# Checklist:

- [x] I have performed a self-review of my own code
- [x] The map tiles are correct and haven't any artefact or missing roads, prefab or other things like this
- [N/A] I've got attached the full map (level 0) on this pull request
- [N/A] I've got attached archive for **Euro truck simulator 2** and **American truck simulator** on this pull request

### CODE REVIEW

**Comment by TheGamer1002** [comment]
CodeQL notes no changes to applicable files

File: README.md [hunk 1]
```diff
@@ -111,37 +109,37 @@ And that's it :D
    how [here](https://www.whatismybrowser.com/detect/what-is-my-local-ip-address))
 2. Open a web browser on your mobile device
 3. Access to the dashboard on your mobile: `http://<YOUR_LOCAL_IP>:3000` (e.g. `http://192.168.1.42:3000`)
-4. And voilà ! You can enable / disable the fullscreen on tap on the dashboard
+4. And voilà! You can enable / disable the fullscreen on tap on the dashboard
 
 > Note: Your mobile device AND your PC need to be on the same network
 
-> Tip: You can save a shortcut on your home screen to use it as an app like.
+> Tip: On an iOS device, you can add the webpage to your home screen like an app! When on the page, tap the Share button, navigate to "Add to home screen," wait ~2 seconds for it to verify, and confirm it. It will go to your home screen and show your neat dashboard shortcut!
```
Comment (JAGFx) [inline README.md:116]: Can you write something for Android or add a generic message for pinning the page to the home screen?

**Review by JAGFx — CHANGES_REQUESTED** [review]
Firstly, many many thanks for your contribution !
Just a little adjustment to do :D

File: README.md (hunk 1 above)
Comment (TheGamer1002) [reply README.md:116]: I'll try. I haven't had any experience with Android, but I assume there is a similar method

**Comment by TheGamer1002** [comment]
Alright, researched a bit and added info for Android. Not sure how to fullscreen it though.

File: README.md (hunk 1 above)
Comment (TheGamer1002) [reply README.md]: Done!

**Comment by TheGamer1002** [comment]
It works in full screen!
20C050A6-01F2-4909-BBF7-71C2B5805263

**Review by JAGFx — APPROVED** [review]

**Pull request merged**